libs/newsletter/guides/Overview.md

```diff
@@ -0,0 +1,67 @@
+# Daffodil Newsletter
+
+The Daffodil Newsletter library manages your newsletter subscription service. It utilizes multiple different ecommerce API drivers to make hooking in your app's newsletter simple while also providing tools to help manage your UI. 
+
+# Getting Started
+This overview assumes that you had already set up your desired daffodil modules. If you have not, first go through the [daffodil installation guide](https://github.com/graycoreio/daffodil/blob/develop/docs/INSTALLATION.md).
```

Comment:
This should reference that you have a Angular project setup and have gone through the `@daffodil/newsletter` installation document with the in-memory-web-api driver.